06-Customizing AzTS for your org/Extending AzTS/Prerequisites.md

```diff
@@ -53,10 +60,10 @@ Above mentioned appsettings can be configured either from Azure portal or using
   - Invoke the configuration cmdlet
     - Configure-CustomControlAdditionPrerequisites -SubscriptionId "AzTSHostingSubscriptionId" -ScanHostRGName "AzTSHostingRGName"
 
------
+
 ## Access to CMET (Control metadata editor tool)
 
-Any user can add new controls using the CMET (Control metadata editor tool) from the AzTS UI. Only the privileged users can access CMET. To elevate any user to privileged role kindly follow the steps mentioned below:
+Only the privileged users can access CMET to update control metadata or to add new control. To elevate any user to privileged role kindly follow the steps mentioned below:
```

Comment:
Only privileged users have access to CMET for updating control metadata or adding new controls. To elevate a user to a privileged role, please follow the steps outlined below:

Reply:
Updated